spec/unit/sibling_decorator_spec.rb

```diff
@@ -23,7 +23,7 @@
             base_path: parent_base_path,
           },
       ],
```

Comment:
I think this line should be indented too.

Reply:
Sure - updated 👍